hibernate-core/src/main/java/org/hibernate/dialect/pagination/LimitHandler.java

```diff
@@ -44,6 +45,18 @@
 	 */
 	String processSql(String sql, RowSelection selection);
 
+	/**
+	 * Return processed SQL query.
+	 *
+	 * @param sql       the SQL query to process.
+	 * @param queryParameters the queryParameters.
```

Comment:
Super minor. It would be great to align both descriptions above vertically.

Reply:
Thanks, I'm going to fix this